src/views/StanzaCal/index.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import * as React from 'react'
+
+const STANZA_CAL_EMBED_URL = 'https://dff2h0hbfv6w4.cloudfront.net/scripts/embed-stanzacal-v1.js'
+
+type PropsType = {};
+
+export default class StanzaCal extends React.Component<PropsType> {
+  componentWillMount() {
+    const script = document.createElement('script')
+    script.setAttribute('src', STANZA_CAL_EMBED_URL)
+    if (document.body) {
+      document.body.appendChild(script)
```

Comment:
Not sure how great of an implementation this is. On the one hand, it creates a cleaner co-location of the StanzaCal script and the StanzaCal component. On the other hand, it causes a bit of a delay in loading the script as compared to just sticking a `<script>` tag directly in `index.html`